test/knex.test.js

```diff
@@ -3,8 +3,9 @@ const Lab = require('@hapi/lab')
 const lab = (exports.lab = Lab.script())
 const { before, describe, it, beforeEach, afterEach } = lab
 const { expect } = require('@hapi/code')
-const Shared = require('seneca-store-test')
+// const Shared = require('seneca-store-test')
```

Comment:
you don't need to comment out all the other tests to run a single test
use `npm run test-some adopt`

Reply:
Indeed, much better